src/components/common/banner/banner.module.css

```diff
@@ -0,0 +1,65 @@
+.wrapper {
+  width: 100%;
+  max-width: 1440px;
+  height: 358px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 auto;
+  margin-bottom: 40px;
+  background-size: cover;
+  background-image: url("../header/../../../assets/brandImages/Group.png");
```

Comment:
why is this url like ../header/../../ ? isnt there a cleaner way like avoiding entering the header folder for example?